### ISSUE TITLE
Broken lines when line-spacing is tweaked

### ISSUE THREAD

**Issue by ianpan870102** [DarthFennec/highlight-indent-guides#69]
I have an additional line-height of 3 pixels, done through `(setq-default line-spacing 3)`. When I use the character-style indent guides, the lines aren't fully connected anymore (with gaps in between lines, as if the vertical character is too short). I understand this is the result of my additional line-height. But is there a way that I can still achieve continuous lines with my line spacing setting?

**Comment by DarthFennec** [comment]
The characters you're using as guides have a fixed height, so if your line height is larger than that you'll get gaps. I've been working on an alternate indent method, `bitmap`, which draws a dynamically-sized image for the guides based on the actual character cell size (see issue #35). I'll try to make sure it works properly with additional line spacing when I release it.

**Comment by DarthFennec** [comment]
Turns out this isn't possible. The `line-spacing` adds an extra gap that I'm not allowed to draw in; I can add the three extra pixels to the guides, but emacs would rather resize the line height to fit as opposed to overflowing the guides into the `line-spacing` gap, so changing the guide height just makes the lines farther apart and never fixes the problem. If there's a way to change this behavior, I can't find it. I checked the `fill-column-indicator` mode and it behaves the same way, and it seems unlikely to me that this problem has a solution that hasn't found its way into that project. So I think you're just out of luck, sorry.
